_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
```

Comment:
Revise: Try to reflect the same message as displayed within the UI. 

For example: 'Learn the art of API blueprint through socialization and collaboration..'